ci/do_ci.sh

```diff
@@ -182,6 +182,16 @@ elif [[ "$1" == "bazel.ipv6_tests" ]]; then
     //test/integration/... //test/common/network/...
   exit 0
 elif [[ "$1" == "bazel.api" ]]; then
+  # verify that all api proto files have correct proto options for java
```

Comment:
Thank you for adding a check for this in CI. 

But I think the correct place to add this type of a check is in tools/check_format.py.

Reply:
Done.